pizza-time/src/components/events/details-request/details-request.js

```diff
@@ -5,13 +5,18 @@ import React, { useState, useEffect } from 'react';
 const Details = (props) => {
   const [data, setData] = useState([]);
   const [loading, setIsLoading] = useState(false);
-  
-  let url = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyA4QgeLdTkXC_ulBAtLDmaY8nEaqZeRtsE&libraries=places&callback=initMap';
+  const [error, setError] = useState('');
+  const API_KEY = process.env.REACT_APP_GOOGLE_PLACES_API_KEY;
+  const url = `https://maps.googleapis.com/maps/api/js?key=${API_KEY}&libraries=places&callback=initMap`;
 
   useEffect(() => {
     renderMap();
   }, [])
 
+  const sendToParent = (req) => {
```

Comment:
Everything looks great here but if I didn't zoom with you I wouldn't know how to use this component. Maybe a quick blurb about what props this takes in.